pkgs/top-level/all-packages.nix

```
@@ -26380,6 +26380,10 @@ with pkgs;
 
   gpsprune = callPackage ../applications/misc/gpsprune { };
 
+  gpu-screen-recorder = callPackage ../applications/video/gpu-screen-recorder/gpu-screen-recorder-cli.nix {};
```

```suggestion
  gpu-screen-recorder = callPackage ../applications/video/gpu-screen-recorder {};
```
and rename the file to default